src/test/java/com/google/firebase/auth/FirebaseUserManagerTest.java

```diff
@@ -344,47 +454,85 @@ public void testInvalidPassword() {
   }
 
   @Test
-  public void testUserUpdater() {
+  public void testUserUpdater() throws IOException {
     UpdateRequest update = new UpdateRequest("test");
+    Map<String, Object> claims = ImmutableMap.<String, Object>of("admin", true, "package", "gold");
     Map<String, Object> map = update
         .setDisplayName("Display Name")
         .setPhotoUrl("http://test.com/example.png")
         .setEmail("test@example.com")
         .setPhoneNumber("+1234567890")
         .setEmailVerified(true)
         .setPassword("secret")
-        .getProperties();
-    assertEquals(7, map.size());
+        .setCustomClaims(claims)
+        .getProperties(Utils.getDefaultJsonFactory());
+    assertEquals(8, map.size());
     assertEquals(update.getUid(), map.get("localId"));
     assertEquals("Display Name", map.get("displayName"));
     assertEquals("http://test.com/example.png", map.get("photoUrl"));
     assertEquals("test@example.com", map.get("email"));
     assertEquals("+1234567890", map.get("phoneNumber"));
     assertTrue((Boolean) map.get("emailVerified"));
     assertEquals("secret", map.get("password"));
+    assertEquals(Utils.getDefaultJsonFactory().toString(claims), map.get("customAttributes"));
+  }
+
+  @Test
+  public void testNullJsonFactory() {
+    UpdateRequest update = new UpdateRequest("test");
+    Map<String, Object> claims = ImmutableMap.<String, Object>of("admin", true, "package", "gold");
+    update.setCustomClaims(claims);
+    try {
+      update.getProperties(null);
+      fail("No error thrown for null JsonFactory");
+    } catch (Exception ignore) {
+      // expected
```

Comment:
Are there any specifics you can verify here?

Reply:
Done. Checking for NPE now.